v3/src/components/graph/models/graph-controller.ts

```diff
@@ -62,22 +62,25 @@ export class GraphController {
   initializeGraph() {
     const {graphModel, layout, dotsRef, enableAnimation, instanceId, v2Document} = this,
       dataConfig = graphModel.config
+
     if (v2Document) {
       this.processV2Document()
     } else {
-      graphModel.setGraphProperties({
-        axes: {bottom: EmptyAxisModel.create({place: 'bottom'}),
+      if (!dotsRef.current){
+        graphModel.setGraphProperties({
```

Comment:
The previous code always called `graphModel.setGraphProperties` whenever we created a `GraphController` without a `v2Document`. I'm not sure what the implications of calling `graphModel.setGraphProperties` conditionally like this might be, so I'll leave that for @bfinzer to determine.

Reply:
It's also not clear to me whether `!dotsRef.current` is the right condition to be testing here (assuming the conditional call is appropriate).

Reply:
I found that once I put the instantiation in a ref (to avoid the original warning), when I let it attempt to `setGraphProperties` without `dotsRef.current`, I would end up with a repeating list of errors for each attempt at each axis:


>```Error: [mobx-state-tree] You are trying to read or write to an object that is no longer part of a state tree. (Object type: 'EmptyAxisModel', Path upon death: '/axes/left', Subpath: 'type', Action: ''). Either detach nodes first, or don't use objects after removing / replacing them in the tree.```